core/deployment/src/main/java/io/quarkus/deployment/ide/IdeConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.deployment.ide;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
+public class IdeConfig {
+
+    /**
+     * The Ide to use to open files from the DevUI.
+     * {@code auto} means that Quarkus will attempt to determine the Ide being used.
+     */
+    @ConfigItem(defaultValue = "auto")
+    public Target target;
+
+    enum Target {
```

Comment:
any way for us to allow user to specify a custom command ?

Reply:
Not yet